### ISSUE TITLE
This repository does not have any remotes hosted on github.com. on linux

### ISSUE THREAD

**Issue by HadrianDiaz** [atom/github#2392]
I'm trying to upload my project files onto a repository on Github that I've already made but there's no option to link it. All the issues open don't have answers or aren't on Linux(I'm actually using the Linux console on a Chromebook.)

### Steps to Reproduce

1.Create project files
2. Goto git tab and select "create repository" 
3. "not on a branch"
4. commit, no remote.

**Expected behavior:**

To ask me for a remote to link so I can upload it to GitHub, or something like that I can't remember how I did it before.

**Actual behavior:**

nothing

**Reproduces how often:**

every time

### Versions

I don't know how to open the terminal for atom but I'm on ChromeOS 80 

### Additional Information

I have no idea what to do.

**Comment by smashwilson** [comment]
Yes, our "initial setup" story has been pretty lacking so far. #2222 and #2329 are some recent work that's somewhat related - with those dialogs, you can _create_ a new repository simultaneously on GitHub and your local filesystem or _publish_ a repository that has no GitHub remotes and have your remotes linked up properly. Those were released in atom/github@0.32.0, which is current bundled in Atom's beta channel (1.44.0-beta0).

That isn't quite what you're specifically asking for here, though - you need a way to _link_ an existing local repository to an existing GitHub repository. I actually started to work on that capability but backed off because that can get messy (do the local repository and the remote one already have divergent histories?)

In the meantime: if it's an option, you could install the [beta channel of Atom](https://atom.io/beta) instead, delete the GitHub repository, and use the "Publish Repository" button that'll be visible on the GitHub tab to recreate it from your local repository. Would that work for you?
